packages/camera/camera/lib/camera.dart

```diff
@@ -433,40 +316,18 @@ class CameraController extends ValueNotifier<CameraValue> {
   /// Throws a [CameraException] if image streaming or video recording has
   /// already started.
   // TODO(bmparr): Add settings for resolution and fps.
-  Future<void> startImageStream(onLatestImageAvailable onAvailable) async {
+  Widget buildView() {
     if (!value.isInitialized || _isDisposed) {
       throw CameraException(
         'Uninitialized CameraController',
-        'startImageStream was called on uninitialized CameraController.',
-      );
-    }
-    if (value.isRecordingVideo) {
-      throw CameraException(
-        'A video recording is already started.',
-        'startImageStream was called while a video is being recorded.',
-      );
-    }
-    if (value.isStreamingImages) {
-      throw CameraException(
-        'A camera has started streaming images.',
-        'startImageStream was called while a camera was streaming images.',
+        'buildView() was called on uninitialized CameraController.',
       );
     }
-
     try {
-      await _channel.invokeMethod<void>('startImageStream');
-      value = value.copyWith(isStreamingImages: true);
+      return CameraPlatform.instance.buildView(_cameraId);
     } on PlatformException catch (e) {
       throw CameraException(e.code, e.message);
     }
```

Comment:
The `try...catch` statement is overkill, simply `return CameraPlatform.instance.buildView(_cameraId)l;` should suffice.